app/apps/handlers.js

```diff
@@ -128,3 +138,31 @@ exports.delete = (req, res, next) => {
     })
     .catch(next);
 };
+
+
+exports.delete_customer = (req, res, next) => {
+  App.get(req.params.id)
+    .then((app) => {
+      app.delete_customer(req.params.id, req.params.customer_id);
+    })
+    .then(() => {
+      res.redirect(req.body.redirect_to);
```

Comment:
No, I don't think so. Should we just redirect straight to the url_for the app page?

Reply:
If it never changes, then I would be inclined to hard-code it here. Otherwise I think we'll need some defensive code to avoid a potential open redirect. I don't think it's actually a vulnerability, because the user has to be signed in to get here, but it's good practice.